docs/esa/hubble/hubble.rst

```diff
@@ -36,39 +34,42 @@ SCIENCE_PRODUCT or POSTCARD).
 This will download all files for the raw calibration level of the observation 'J6FL25S4Q' and it will store them in a tar called
 'raw_data_for_J6FL25S4Q.tar'.
 
-.. code-block:: python
+.. doctest-remote-data::
 
-  >>> from astroquery.esa.hubble import ESAHubble
-  >>> esahubble = ESAHubble()
-  >>> esahubble.download_product(observation_id="J6FL25S4Q", calibration_level="RAW", filename="raw_data_for_J6FL25S4Q.tar")
+  >>> from astroquery.esa.hubble import ESAHubble   # doctest: +IGNORE_OUTPUT
+  >>> esahubble = ESAHubble()   # doctest: +IGNORE_OUTPUT
```

Comment:
There should be no outputs here, why the comments?

Reply:
here and everywhere below

Reply:
I.... have no explanation, I thought I removed all the ones where there was no output but apparently not.